updatedb.py

```diff
@@ -0,0 +1,46 @@
+import json
+import csv
+
+
+with open('database.json') as file:
+    db = json.loads(file.read())
+
+db_id = len(db)+1
+db_names = [person['name'] for person in db]
+next_round_ids = []
+
+# update the database with the round 5 groupings
+def update_db(db, filename):
+    with open(filename) as file:
+        groups = json.loads(file.read())
+        for row in groups:
+            for id in row['group']:
+                db[id-1]['connections'] += row['group']
+                db[id-1]['connections'].remove(id)
+    return db
+
+
+with open('round6.csv') as file:
+    reader = csv.reader(file, delimiter=',')
+    for i, row in enumerate(reader):
+        if i > 0:
```

Comment:
is this just a check if it's the first row (headers)? there might be some useful props on `reader`; looks like it at least has `fieldnames`: https://docs.python.org/3/library/csv.html#reader-objects

maybe we don't need that though; i'm not sure what this is doing